packages/core/src/part.js

```diff
@@ -357,20 +357,68 @@ Part.prototype.generateTransform = function(transforms) {
   }
 }
 
-/** Chainable way to set the grain property */
-Part.prototype.setGrain = function (grain = 90) {
-  this.attributes.set('data-grain', grain)
+/** Chainable way to add the cut info */
+Part.prototype.addCut = function (cut=2, material='fabric', identical=false) {
+  if (cut === false) {
+    if (this.cut.materials[material]) delete this.cut.materials[material]
+    else this.context.raise.error(`Tried to remove a material that is not set`)
+    return this
+  }
+  if (typeof this.cut.materials[material] === 'undefined') this.cut.materials[material] = {}
+  if (typeof cut !== 'number') {
+    this.context.raise.error(`Tried to set cut to a value that is not a number`)
+    return this
+  }
+  if (typeof material !== 'string') {
+    this.context.raise.error(`Tried to set material to a value that is not a string`)
+    return this
+  }
+  this.cut.materials[material].cut = cut
+  this.cut.materials[material].identical = identical
+
+  return this
+}
+
+/** Chainable way to remove (some) cut info */
+Part.prototype.removeCut = function (material=false) {
+  if (!material) {
+    this.context.raise.warning('Called part.removeCut() without any parameters. Not removing anything')
+    return this
+  }
+  if (typeof material !== 'string') {
+    this.context.raise.error(`Tried to set material to a value that is not a string`)
+    return this
+  }
+  if (this.cut.materials[material]) delete this.cut.materials[material]
+  else this.context.raise.error(`Tried to remove a material that is not set`)
 
   return this
```

Comment:
This should internally call `this.setCut(false, material)` rather than doing any of its own business